recipe/meta.yaml

```diff
@@ -10,24 +10,22 @@ package:
 
 source:
   url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: c93fc0ed8216c71f9b8264788ca7743e26f4498b2011e5e2404696e5fd238e68
+  sha256: c8e9d769a6066bb5f94bb0362dbc20ed6f6b525336373263aeb6f9c6dc7b91e4
 
 build:
   number: 0
   skip: true  # [py<37 or osx]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}  # [win]
+    - {{ compiler('cxx') }}  # [win]
+    - {{ compiler('c') }} =11.2  # [not win]
+    - {{ compiler('cxx') }} =11.2  # [not win]
     - sysroot_linux-64 2.17  # [linux64]
   host:
     - setuptools
     - cython
-    - pycparser
-    - autopxd2
-    - pyclibrary
     - python
     - pip
     - cudatoolkit {{ major_version ~ ".*.*" }}
```

Comment:
Could you remind me what this means? `{{ major_version ~ ".*.*" }}` If we need to use headers from the same CUDA major.minor.patch version, shouldn't we lock it here, and relax the pin for the run section?

Reply:
just double checked, we need to use the headers from the same major.minor, patch version can still be wildcard, so updated pin on the build would be `{{ major_version ~ "." ~ minor_version ~ ".*" }}`

I should probably switch this to a pinning expression at some point.

Reply:
I doubt it'd work, conda-forge maintains a single source of truth for the CUDA headers by using the right version of the docker image, perhaps it's better to add 11.7.1 image first...? cc: @jakirkham for vis

Reply:
Wait, so CUDA Python 11.7.1 does not map to CUDA 11.7.1?! @m3vaz

Reply:
> Wait, so CUDA Python 11.7.1 does not map to CUDA 11.7.1?! 

Despite my whines about this offline, it's actually simpler, as we have already the CUDA 11.7.0 image ready for use. I'm making the change locally and will push.

Reply:
Fixed in bed42e458dd4e603494a528dd37be9cc660fe144.